content/blog/2018-05-10-onboarding-social-weather.md

```diff
@@ -0,0 +1,690 @@
+</tr>
+<tr class="odd">
+<td style="text-align: left;">#### documentation</td>
+</tr>
+<tr class="even">
+<td style="text-align: left;">the package includes all the following forms of documentation:</td>
+</tr>
+<tr class="odd">
+<td style="text-align: left;">- [x] <strong>a statement of need</strong> clearly stating problems the software is designed to solve and its target audience in readme</td>
+</tr>
+<tr class="even">
+<td style="text-align: left;">- [x] <strong>installation instructions:</strong> for the development version of package and any non-standard dependencies in readme</td>
+</tr>
+</tbody>
+</table>
+
+Now if we clean up a bit…
+
+``` r
+threads <- dplyr::group_by(threads, line)
+threads <- dplyr::mutate(threads, no_total_occ = n(),
+                     no_issues = length(unique(issue)), 
+                     size = stringr::str_length(line))
+threads <- dplyr::ungroup(threads)
+threads <- dplyr::group_by(threads, issue, line)
+threads <- dplyr::arrange(threads, created_at)
+threads <- dplyr::filter(threads, no_total_occ <= 2,
+                     # for repetitions in total keep the short ones 
+                     # bc they are stuff like "thanks" so not template
+                     # yes 10 is arbitrary
+                     no_issues <= 1 | size < 10)
+# when there's a duplicate in one issue
+# it's probably citation
+# so keep the first occurrence
+get_first <- function(x){
+  x[1]
+}
+threads <- dplyr::group_by(threads, issue, line)
+threads <- dplyr::summarise_all(threads, get_first)
+threads <- dplyr::select(threads, - no_total_occ, - size, - no_issues)
+
+threads <- dplyr::mutate(threads, # let code words now be real words
+                     line = stringr::str_replace_all(line, "`", ""),
+                     # only keep text from links, not the links themselves
+                     line = stringr::str_replace_all(line, "\\]\\(.*\\)", ""),
+                     line = stringr::str_replace_all(line, "\\[", ""),
+                     line = stringr::str_replace_all(line, "blob\\/master", ""), 
+                     # ’
+                     line = stringr::str_replace_all(line, "’", ""), 
+                     # remove some other links
+                     line = stringr::str_replace_all(line, "https\\:\\/\\/github\\.com\\/", ""))
+
+threads <- dplyr::filter(threads, !stringr::str_detect(line, "estimated hours spent reviewing"))
+threads <- dplyr::filter(threads, !stringr::str_detect(line, "notifications@github\\.com"))
+threads <- dplyr::filter(threads, !stringr::str_detect(line, "reply to this email directly, view it on"))
+threads <- dplyr::ungroup(threads)
+```
+
+Here is what we get from the same review.
+
+``` r
+dplyr::filter(threads, user == "jsta", is_review)  %>%
+  dplyr::select(line) %>%
+  head() %>%
+  knitr::kable()
+```
+
+<table>
+<colgroup>
+<col style="width: 100%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th style="text-align: left;">line</th>
```

Comment:
The separation between the rendered output here and your blog text that you wrote is not super clear - maybe putting `<br><br>` in to make a few line breaks, or something else?

same for all other similar cases 

![screen shot 2018-05-02 at 9 44 27 am](https://user-images.githubusercontent.com/577668/39537138-82a32f9e-4ded-11e8-8ff4-b4438383b3cc.png)